packages/labs/eleventy-plugin-lit/src/index.ts

```diff
@@ -27,9 +27,14 @@ module.exports = {
           const html = (await import('lit')).html;
           const unsafeHTML = (await import('lit/directives/unsafe-html.js'))
             .unsafeHTML;
-          for (const module of options.componentModules ?? []) {
-            require(path.join(process.cwd(), module));
-          }
+          // TODO(aomarks) This doesn't work in Eleventy --watch mode, because
+          // the ES module cache never gets cleared. Switch to isolated VM
+          // modules so that we can control this.
+          await Promise.all(
+            (options.componentModules ?? []).map(
+              (module) => import(path.join(process.cwd(), module))
```

Comment:
> You might want path.resolve() here?

Done. (Because of canonicalization? I'm not sure if that affects the module identity for the cache, but maybe?)

> For security, might you want to also make sure that the path is in the project root? Or are there cases you want to support reaching out of the cwd?

I don't think we can really define what the project root is (cwd?), and I don't really see why we shouldn't be able to reach of it anyway. In general, an eleventy build config can access any files it wants to that the OS lets it. Why should we introduce a boundary?